### DISCUSSION TITLE
GPS location on LiveMap using smartphone GPS receiver

### DISCUSSION THREAD

**Discussion by radzik80** [dl9rdz/rdz_ttgo_sonde#252] [Q&A] [answered]
Hello, 

Is there any possibility or future plans to use smartphone GPS receiver to be uses as receiver location sensor? 
Smartphone in hand probably has more accuracy readings than TTGO hidden in backpack or pocket.

**Comment by LukePrior** [comment]
This is already a feature when using the app.

**Reply by radzik80** [reply]
I meant LiveMap using web browser as iOS users can't install rdzwx-go app.
Is there any way to install app on iOS device?

**Comment by dl9rdz** [comment, marked as answer]
No.

LiveMap is http, not https.
No space in code to support https
Geolocation API access is not permitted on "non-secure" http web pages on most recent web browsers (e.g. on ChromeOS since 2016, on iOS since Safari 10 (i.e. 2017)).
(This was one of the main reasons I put my JavaScript code into an app and not a http page on the TTGO in the first place)

**Reply by radzik80** [reply]
All clear, thanks for explanation.
